base_layer/core/src/core_key_manager/interface.rs

```diff
@@ -37,13 +37,21 @@ use crate::transactions::{
     tari_amount::MicroTari,
     transaction_components::{
         EncryptedData,
+        KernelFeatures,
+        RangeProofType,
         TransactionError,
         TransactionInputVersion,
         TransactionKernelVersion,
         TransactionOutputVersion,
     },
 };
 
+#[derive(Clone, Copy, PartialEq)]
+pub enum TxoType {
+    Input,
+    Output,
+}
+
 #[derive(Clone, Copy, EnumIter)]
 pub enum CoreKeyManagerBranch {
     DataEncryption,
```

Comment:
Not part of this PR, I know, but this field name is vague and seems like it is general purpose

Reply:
Same goes for "Nonce"